packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -722,7 +766,8 @@ export class IntervalCollectionView<TInterval extends ISerializableInterval> ext
         if (interval) {
             // Local ops get submitted to the server. Remote ops have the deserializer run.
             if (local) {
-                this.emitter.emit("add", undefined, serializedInterval);
+                // Review: Is this case possible?
+                this.emitter.emit("delete", undefined, serializedInterval);
             } else {
```

Comment:
i don't understand this case? why send a delete op?

Reply:
Oh, that just looks like a bad merge. I'll fix it.